### PR TITLE
feat: add migrations from v1.9.x to v2.x

### PR DESCRIPTION
**Pull request by paul-nicolas**
Fixes NUM-1643

### CODE REVIEW

File: components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go [hunk 1]
```diff
@@ -0,0 +1,514 @@
...
+
+var (
+	batchSize             uint64 = 10000
+	oldSchemaRenameSuffix        = "_save_v2_0_0"
+)
+
+func InitMigrationConfigCLIFlags(flags *flag.FlagSet) {
+	flags.Uint64(LogsMigrationBatchSizeFlag, 10000, "Batch size for logs migration")
+	flags.String(OldSchemaRenameSuffixFlag, "save_v2_0_0", "Name of the old schema (to be renamed)")
+
```
Comment (gfyrag) [inline components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go:43]: Flags are not bound at this point ;)

File: components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go [hunk 2]
```diff
@@ -0,0 +1,514 @@
...
+	rows, err := sqlTx.QueryContext(ctx, sb.String())
+	if err != nil {
+		if err == sql.ErrNoRows {
+			return rawLogs, nil
+		}
+
+		return nil, errors.Wrap(err, "selecting logs")
+	}
+
```
Comment (gfyrag) [inline components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go:112]: Maybe add a defer to close rows if the loop fail.

Comment (paul-nicolas) [reply components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go:110]: Done thanks !

File: components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go (hunk 1 above)
Comment (paul-nicolas) [reply components/ledger/pkg/storage/sqlstorage/ledger/migrates/0-init-schema/any.go]: Done thanks !

**Review by gfyrag — APPROVED** [review]

**Pull request merged**